backend/library/views.py

```
@@ -1,6 +1,9 @@
+from itertools import chain
 import json
 from django.db import IntegrityError
-from rest_framework import viewsets
+from django.db.models import F, Q, IntegerField, OuterRef, Subquery
+from rest_framework import viewsets, status
```

Ruff (0.8.2)

`rest_framework.viewsets` imported but unused

Remove unused import

(F401)